esi_leap/common/utils.py

```diff
@@ -19,3 +20,9 @@
 
 def get_resource_lock_name(resource_type, resource_uuid):
     return resource_type + "-" + resource_uuid
+
+
+def datetime_aware(dt):
+    if dt.tzinfo is not None and dt.tzinfo.utcoffset(dt) is not None:
```

Comment:
When will `dt.tzinfo.utcoffset(dt)` return `None` when `dt.tzinfo` is not None?

Reply:
Not entirely sure myself, but the python documentation implies it can be:

https://docs.python.org/3/library/datetime.html#determining-if-an-object-is-aware-or-naive